src/inputpipe.c

```diff
@@ -59,12 +52,10 @@ int inputpipe_handle(inputpipe_ctx *ctx) {
 	struct timespec ctime;
 	obtainsystime(&ctime);
 
-	// do not over-read
-	struct timespec readuntil = timeAddMs(&ctime, snapctx.bufferms * 5 / 4);  // target 80% buffer fill-rate
-	bool buffer_full = (timespec_cmp(ctx->lastchunk, readuntil) > 0);
+	struct timespec start_playing_at = timeAddMs(&ctime, snapctx.bufferms * 9 / 10);  // target 90% buffer fill-rate to allow for some negative clock drift.
```

Comment:
If we set start_playing_at to something very close to the current timestamp then we can use large buffers and still start playback when hitting "play" instead of bufferMS milli seconds later.